KsApi/models/graphql/adapters/Project+FetchProjectQueryDataTests.swift

```diff
@@ -44,6 +44,7 @@ final class Project_FetchProjectQueryDataTests: XCTestCase {
     XCTAssertFalse(project.displayPrelaunch!)
     XCTAssertTrue(project.prelaunchActivated!)
     XCTAssertEqual(project.watchesCount, 180)
+    XCTAssertEqual(project.isPledgeOverTimeAllowed, true)
```

Comment:
nit: you could just `XCTAssertTrue` here as an alternative. No biggie either way!

Reply:
Good catch